scripts/GameCorner.asm

```diff
@@ -117,48 +120,54 @@ CeladonGameCornerScript2:
 	ret
 
 GameCorner_TextPointers:
-	dw CeladonGameCornerText1
-	dw CeladonGameCornerText2
-	dw CeladonGameCornerText3
-	dw CeladonGameCornerText4
-	dw CeladonGameCornerText5
-	dw CeladonGameCornerText6
-	dw CeladonGameCornerText7
-	dw CeladonGameCornerText8
-	dw CeladonGameCornerText9
-	dw CeladonGameCornerText10
-	dw CeladonGameCornerText11
-	dw CeladonGameCornerText12
-	dw CeladonGameCornerText13
-
-CeladonGameCornerText1:
-	text_far _CeladonGameCornerText1
+	dw CeladonGameCornerText_NPC_CanExchangePrizesNextDoor
+	dw CeladonGameCornerText_NPC_CoinSalesman
+	dw CeladonGameCornerText_NPC_KeepThisQuiet
+	dw CeladonGameCornerText_NPC_MachineDiffOdds
+	dw CeladonGameCornerText_NPC_10Coins
+	dw CeladonGameCornerText_NPC_WonderfulTime
+	dw CeladonGameCornerText_NPC_GymGuy
+	dw CeladonGameCornerText_NPC_GamesAreScary
+	dw CeladonGameCornerText_NPC_20Coins_2
```

Comment:
What's the reason the first occurrence is the one named with `_2`?

Reply:
As commented on #331, the text-table and script-table labels may as well stay with their IDs in the names. Identifying the labels that end in their own addresses is still a nice improvement.